validation/validation_test.go

```diff
@@ -37,10 +37,11 @@ func testValidation(t *testing.T, chain *ChainConfig) {
 // designed to protect downstream software or
 // sanity checking basic consistency conditions.
 func testUniversal(t *testing.T, chain *ChainConfig) {
-	t.Run("Genesis Hash Check", func(t *testing.T) { testGenesisHash(t, chain.ChainID) })
-	t.Run("Genesis RPC Check", func(t *testing.T) { testGenesisHashAgainstRPC(t, chain) })
-	t.Run("Uniqueness Check", func(t *testing.T) { testIsGloballyUnique(t, chain) })
-	t.Run("ChainID RPC Check", func(t *testing.T) { testChainIDFromRPC(t, chain) })
+	t.Run("Genesis Hash", func(t *testing.T) { testGenesisHash(t, chain.ChainID) })
+	t.Run("Genesis RPC", func(t *testing.T) { testGenesisHashAgainstRPC(t, chain) })
+	t.Run("Uniqueness", func(t *testing.T) { testIsGloballyUnique(t, chain) })
+	t.Run("ChainID RPC", func(t *testing.T) { testChainIDFromRPC(t, chain) })
+	t.Run("Optimism Config", func(t *testing.T) { testOptimismConfig(t, chain) })
```

Comment:
If these names are changed, we need to update the matchers in `exclusions_test.go`. We could also try to make that less brittle by defining some constants.

Reply:
Defined some constants since these string literals are used in multiple places